.drone/drone.yml

```diff
@@ -82,7 +82,7 @@ trigger:
 
 steps:
   - name: lint
-    image: grafana/agent-build-image:0.17.0
```

Comment:
Can you rebase against main? There's a 0.19.0 now so we wouldn't want to update these here.

Reply:
This should be all set now